database/src/main/resources/db/migration/V1_147__all_companies_again.sql

```diff
@@ -0,0 +1,92 @@
+-- Use UNION to take only unique companies (name,id,created,source) into account.
```

Comment:
This could use a comment about what the view is for and what it does. Something along the lines of, (i don't know if this is correct) "exposes company information from 3 different sources: 1) ckan group tables joined to transport-operator table 2) associated-services table (read from csv and not registered in NAP service), 3) transport-operator defined companies not registered in ckan"

Reply:
Comment updated